src/frontend/components/ErrorComponents/FullScreenError.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { ErrorComponentsProps, FullScreenError } from './index';
+
+// 👇 This default export determines where your story goes in the story list
+export default {
+  title: 'Components/FullScreenError',
+  component: FullScreenError,
+};
+
+// 👇 We create a “template” of how args map to rendering
+const Template = (
+  args: JSX.IntrinsicAttributes &
+    ErrorComponentsProps & { children?: React.ReactNode },
+) => <FullScreenError {...args} />;
```

Comment:
Maybe I missed something, but this seems ok and allow to remove `// @ts-ignore`
```suggestion
const Template: Story<ComponentProps<typeof FullScreenError>> = (args) => (
  <FullScreenError {...args} />
);
```

Reply:
This was added by my IDE, to be honest I'm struggling to understand this kind of syntax :/
I'll try your suggestion.

Reply:
Works great, thanks :pray: